backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/llvm/coverage/CoverageManager.kt

```diff
@@ -27,9 +27,7 @@ internal class CoverageManager(val context: Context) {
             context.config.configuration.getBoolean(KonanConfigKeys.COVERAGE)
 
     private val librariesToCover: Set<String> =
-            context.config.configuration.getList(KonanConfigKeys.LIBRARIES_TO_COVER)
-                    .map { File(it).absolutePath.removeSuffixIfPresent(".klib") }
-                    .toSet()
+            context.config.coveredLibraries.map { it.libraryName }.toSet()
```

Comment:
Shouldn't source libraries be added here as well?

Reply:
I discussed it with @sbogolepov. The final decision is that "source" and "covered" are orthogonal properties of libraries and they should be specified separately.